src/ui/page/share/dataset/steps/serviceStorageSelection/serviceStorageSelection.scss

```
@@ -37,12 +37,47 @@
       max-height: 610px;
       overflow-y: auto;
 
+      &.detailed {
+        @include bi-column-item(left);
+      }
+
       .okp4-dataverse-portal-share-dataset-page-services {
         display: grid;
         grid-template-columns: repeat(auto-fill, minmax(260px, 1fr));
         grid-auto-rows: 220px;
         gap: 48px;
         margin-right: 15px;
+
+        .okp4-dataverse-portal-share-dataset-page-service {
+          :focus-visible {
+            outline: none;
+          }
+
+          &.selected {
+            border: 2px solid themed('primary-color');
+            background-color: themed('card-background-selected');
+          }
```

In order to avoid layout shifting on click when adding a border, you can define a transparent background by default:
`border: 2px solid transparent;`

However, using the outline props with a negative offset is the most appropriate alternative, because it doesn't affect the layout!
```suggestion
        .okp4-dataverse-portal-share-dataset-page-service {
          :focus-visible {
            outline: none;
          }

          &.selected {
            outline: 2px solid themed('primary-color');
            outline-offset: -2px;
            background-color: themed('card-background-selected');
          }
```

Also, the service cards missing visual accessibility for a keyboard user.
We might probably want to provide an outline props to the service that has a `focus:visible` button:

```scss
.okp4-dataverse-portal-share-dataset-page-service:has(:focus-visible) {
    outline: 2px solid themed('primary-color');
    outline-offset: -2px;
}
```